### PR TITLE
Add a bit more instrumentation metadata

### PR DESCRIPTION
**Pull request by cd1m0**
This PR adds the following new info to the instrumentation metadata:

- text of the original annotation
- name of each variable in the debug event encoding

### CODE REVIEW

File: test/multifile_samples/macros/instrumentationMetadata.json.expected [hunk 1]
```diff
@@ -274,11 +274,13 @@
         "154:37:2",
         "72:21:0"
       ],
+      "annotationText": "OwnershipTransferred(address indexed ",
```
Comment (blitz-1306) [inline test/multifile_samples/macros/instrumentationMetadata.json.expected:277]: This also looks like a fragment

File: test/multifile_samples/erc20_macro_inheritance/instrumentationMetadata.json.expected [hunk 2]
```diff
@@ -1733,11 +1733,13 @@
         "302:49:2",
         "35:48:0"
       ],
+      "annotationText": "ate);\n    }\n }",
```
[the remark this answers is not in the document]
Comment (cd1m0) [reply test/multifile_samples/erc20_macro_inheritance/instrumentationMetadata.json.expected]: Good catch. The code wasn't correctly computing the original text for macro instantiations. Fixed in [fcd2351](https://github.com/ConsenSys/scribble/pull/179/commits/fcd235144e433fcac6ec0791439313e222dcf60f).

File: test/multifile_samples/macros/instrumentationMetadata.json.expected (hunk 1 above)
Comment (cd1m0) [reply test/multifile_samples/macros/instrumentationMetadata.json.expected]: Good catch. The code wasn't correctly computing the original text for macro instantiations. Fixed in [fcd2351](https://github.com/ConsenSys/scribble/pull/179/commits/fcd235144e433fcac6ec0791439313e222dcf60f).

**Review by blitz-1306 — APPROVED** [review]
Looks good to merge

**Pull request merged**